wordpress-mobile/WordPress-Android
# Issue/13268 bind ui

Partially fixes #13268 

This PR maps data provided by the VM to the UI. 

Merge instructions:
1. Make sure https://github.com/wordpress-mobile/WordPress-Android/pull/13450 is merged
2. Remove "Not ready for merge" label
3. Merge this PR

Notes:
- The final designs are not ready yet, so the UI is not polished. 
- On item click logic is implemented in the next PR


To test:
- Make sure the ActivityLogFiltersFeatureConfig is enabled
1. My site -> Activity Log
2. Click on "Activity Type" chip
3. Notice a loading screen is shown
4. Notice the loading screen gets replaced with the content screen after 1s

PR submission checklist:
